### PR TITLE
[MINI-6189] - Send Analytics to Miniapp

### PR DESCRIPTION
**Pull request by rleojoseph**
# Description
* Miniapp can send analytics using this new interface

## Links
[MINI-6189](https://jira.rakuten-it.com/jira/browse/MINI-6189)

# Checklist
- [x] I have completed all steps in the [Contribution Checklist](../blob/master.github/CONTRIBUTING.md#checklist)

### CODE REVIEW

**Review by GondiTilak — APPROVED** [review]
Looks good to me :+1:

File: js-miniapp-bridge/src/common-bridge.ts [hunk 1]
```diff
@@ -811,3 +821,10 @@ function trimBannerText(message: string = null, maxLength = 128) {
     ? message?.substring(0, maxLength - 1) + '…'
     : message;
 }
+
+function BooleanValue(value) {
+  if (value.toLowerCase() === 'true') {
+    return true;
+  }
+  return false;
+}
```
Comment (Climbatize) [inline js-miniapp-bridge/src/common-bridge.ts:830]: I would suggest a more flexible function:

```suggestion
function BooleanValue(value) {
  if (typeof value === 'boolean') {
    // If the value is already a boolean, return it
    return value;
  } else if (typeof value === 'string') {
    // If the value is a string, check for common representations of boolean values
    const lowerCaseValue = value.toLowerCase();
    if (lowerCaseValue === 'true' || lowerCaseValue === '1') {
      return true;
    } else if (lowerCaseValue === 'false' || lowerCaseValue === '0') {
      return false;
    }
  } else if (typeof value === 'number') {
    // If the value is a number, check for integer representations of boolean values
    if (value === 1) {
      return true;
    } else if (value === 0) {
      return false;
    }
  }

  // If the value cannot be interpreted as a boolean, return false
  return false;
}
```

unit tests:
```javascript
console.log(BooleanValue(true)); // true
console.log(BooleanValue(false)); // false
console.log(BooleanValue('true')); // true
console.log(BooleanValue('false')); // false
console.log(BooleanValue('1')); // true
console.log(BooleanValue('0')); // false
console.log(BooleanValue(1)); // true
console.log(BooleanValue(0)); // false
console.log(BooleanValue(null)); // false
console.log(BooleanValue('hello')); // false
console.log(BooleanValue(42)); // false
```

**Review by Climbatize — APPROVED** [review]
LGTM. See my suggestion

File: js-miniapp-bridge/src/common-bridge.ts (hunk 1 above)
Comment (rleojoseph) [reply js-miniapp-bridge/src/common-bridge.ts]: Thanks for the feedback. We don't need to take care of number types though, I have updated the function now

**Review by munir-rakuten — COMMENTED** [review]
looks good to me, thanks.

**Pull request merged**